src/mpid/ch4/netmod/ofi/ofi_init.c

```diff
@@ -679,15 +687,13 @@ int MPIDI_OFI_init_local(int *tag_bits)
 
     MPIDI_OFI_global.num_vnis = num_vnis;
```

Comment:
Remove this line and remove the variable `num_vnis`?

Reply:
Or revert the changes above? Not sure why I have those...